dbt_schema_builder/builder.py

```diff
@@ -10,7 +10,8 @@
 import yaml
 from dbt.config import RuntimeConfig
 from dbt.exceptions import DatabaseException
-from dbt.logger import GLOBAL_LOGGER as logger
+from dbt.events import AdapterLogger
```

Comment:
Do we still need the `log_manager` if you are setting the logger? Also, is the the logger variable used anywhere?

Reply:
`log_manager` is used here: [https://github.com/edx/dbt-schema-builder/blob/main/dbt_schema_builder/builder.py#L611](https://github.com/edx/dbt-schema-builder/blob/main/dbt_schema_builder/builder.py#L611)
but for rest of the places Snowflake AdapterLogger object `logger` is used.